### DISCUSSION TITLE
Catalogue entry image zoom not resetting to normal size when closed

### DISCUSSION THREAD

**Discussion by zsofiaj** [thegetty/quire#778] [Feedback]
### Before proceeding, check to make sure there isn’t an existing issue for this bug.

- [X] I have searched the existing issues and determined this is a new bug.

### Steps to reproduce

When I click on the little square zoom symbol at the top left of the main image on a catalogue entry, I zoom in on full-screen image, and then I close the zoomed lightbox view, the image shows up partially zoomed and sometimes misaligned on the main entry page. Even if I zoom out in the Lightbox view, sometimes I can't get the image back to the center unless I reload the page. Is there a way to make that image reset to the original size every time the lightbox view is closed? Maybe this is a known issue that you are working on, but I wasn't able to find it in this list of issues. I've uploaded 4 screenshots here of the sequence of opening the zoomed image, zooming, then closing it.

<img width="1433" alt="Screen Shot 2023-05-08 at 3 51 42 PM" src="https://user-images.githubusercontent.com/68071544/236920356-9cf173ec-321c-45b8-ad25-7d97f2458f24.png">

<img width="1440" alt="Screen Shot 2023-05-08 at 3 51 50 PM" src="https://user-images.githubusercontent.com/68071544/236920359-6be19c10-b611-4c98-8a55-bbf9750a91e9.png">

<img width="1439" alt="Screen Shot 2023-05-08 at 3 52 03 PM" src="https://user-images.githubusercontent.com/68071544/236920365-e41506c9-97dc-451c-b9a6-7a00c35eade4.png">

<img width="1429" alt="Screen Shot 2023-05-08 at 3 52 12 PM" src="https://user-images.githubusercontent.com/68071544/236920373-f03eee10-8160-48f9-b7da-8f028836f16f.png">

### Actual behavior

Zoomed image stays zoomed and off to the side if that is the area that was zoomed in the lightbox view.

### Expected behavior

I would expect the image to reset to the normal size and layout when I close the lightbox view. 

### Version numbers

MacOS 11.5.1
Chrome 109.0.5414.119
Safari 14.1.2
Quire version 1.0.0-rc.3

### What browsers are you seeing the problem on?

Chrome, Safari

**Comment by Erin-Cecele** [comment]
I am able to recreate this issue in the starter project.

MacOS: Monterey 12.5.1
Chrome: 113.0.5672.63
Firefox: 112.0.2
Quire version 1.0.0-rc-6

The image does not automatically go back to its original state after zooming in via the lightbox. It takes some fidgeting to zoom out and recenter to get it close to the original.

**Comment by Erin-Cecele** [comment]
@zsofiaj - Thank you for bringing this behavior to our attention in issue #723. We have now closed that issue and moved it to the discussion board because it is not actually a bug but rather Quire operating as intended. Because of the way annotated images work, once an image is zoomed in on, we want it to retain its state so a person can go back and forth between the text and the image. 

That said, we would be eager to hear from you and other members of the community whether there is a strong case to change this behavior or maybe have an option to "restore" the image to its original state. Thank you in advance for your feedback!

**Comment by Erin-Cecele** [comment]
Per issue #723 this issue has been resolved
